Makefile

```diff
@@ -69,7 +69,7 @@ uninstall: uninstall-base uninstall-man uninstall-doc uninstall-$(type)
 
 install-base:
 	@echo "Installing scripts..."
-	@sed -i.orig 's|@LIBDIR@|$(DESTDIR)$(LIBDIR)|' src/recap
+	@sed -i.orig 's|\(declare.* LIBDIR=\).*|\1"$(DESTDIR)$(LIBDIR)/recap"|' src/recap
```

Comment:
Can you add the full regex for the line, i.e. include terminators `^$`.

Reply:
In general there are a couple of reasons to use them, is better to be `explicit` vs `implicit`. Also using them improves performance, using anchors improve on the matching for those cases where the string is not matching from the very beginning jumping almost immediately to the next line without having to match the whole string.

This article has a good example of anchors https://www.loggly.com/blog/five-invaluable-techniques-to-improve-regex-performance/

Reply:
I agree with you that being more explicit is best, and it definitely makes a difference for performance. I was thinking that you found scenarios that I didn't account for. I was thinking about it from an accuracy standpoint and couldn't see how adding the anchors matched scenarios that I hadn't already thought about.

Thanks for the article and for the context. This highlights the need that I should get into the habit of always writing regex according to best practices, instead of only doing so when necessary. 

(Best) Practice makes perfect. :smile: